docs/context-specification.md

```diff
@@ -53,4 +53,60 @@ _Each goal, whether it is a business goal, a usage goal, or a system goal, is is
 | | Researcher | Identifying collaborators with specific capabilities to answer a funding call | Usage |
 | | Researcher | Visualizing a personal capability portfolio | Business |
 | | Institution | Visualizing success of own research groups | Business |
-| | Funding Agency | Assessing the academic profile of a researcher | Usage |
\ No newline at end of file
+| | Funding Agency | Assessing the academic profile of a researcher | Usage |
+
+## Domain Model
+
+```mermaid
+classDiagram
+    direction LR
+
+    class ResearchGroup {
+
+    }
+
+    class LeadResearcher {
+
+    }
+
+    class CapabilityPortfolio {
+
+    }
+
+    class Researcher {
+        +name
+    }
+
+    class Institution {
+        +name
+    }
+
+    class Capability {
+        +name
+        +definition
+        +strenghtOfEvidence
+    }
+
+    class Evidence {
```

Comment:
This class could benefit from further sub-classes (e.g., repository, tool, certificate, publication)